packages/vscode-extension/src/commands/ManageAccounts.ts

```diff
@@ -141,14 +141,17 @@ export default class ManageAccounts extends BaseAction {
 
 	private async handleBrowserAuth(accountCredentialsList: AuthListData) {
 		const authId = await this.getNewAuthId(accountCredentialsList);
-		const url = await this.getUrl();
-		if (!authId) {
+		if (!authId || authId === undefined) {
```

Comment:
!authId already checks for null, undefined, and empty string. So for this one, I think !authId should be enough.